content/Hardware/Generic/About-the-AREF-pin.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "About the AREF pin"
+id: 360018922239
+---
+
+
+The **AREF (Analog Reference) pin** can be used to provide an external **reference voltage** for the analog-to-digital conversion of inputs to the analog pins. The reference voltage essentially specifies the value for the top of the input range, and consequently each discrete step in the converted output.
+
+Advantages of using the AREF pin include:
+
+* Precise control of reference value (instead of a limited number of built-in options).
+* No programming logic required to adjust reference voltage.
+* In some cases, a less noisy signal.
+
+Not all boards have an enabled AREF pin, and it doesn't work the same way on all boards that do:
+
+<table>
+  <tr>
+    <td style="width:50%">
+      <ul>
+        <li>Arduino AVR Boards (Uno, Mega, Leonardo, etc.)</li>
+        <li>Arduino SAMD Boards (Zero, etc.)</li>
+        <li>Arduino megaAVR Boards (Uno WiFi Rev2)</li>
+      </ul>
+    <td>External reference voltage can be read from the AREF pin. Enable it by calling <a href="https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/">analogreference()</a> with the appropriate option for your board, i.e. <code>analogReference(EXTERNAL)</code> for AVR boards.</td>
```

Comment:
```suggestion
    <td>External reference voltage can be read from the AREF pin. Enable it by calling <a href="https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/">analogreference()</a> with the appropriate option for your board, i.e. <code>analogReference(EXTERNAL)</code> for AVR boards. > Make sure to follow the <a href="https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/#_notes_and_warnings">Notes and Warnings</a> section to avoid damaging the microcontroller on the Arduino board </td>
```

Since there is a possibility for shorting the board while using AREF, I think that we should make sure that the users read the Notes and Warnings section.

Reply:
Good point. `>` is Markdown syntax and won't work inside HTML but I addressed your feedback in 5a6b62e8d51aaefc2390eeb4fca92afc982210ac.